instagram/src/components/Post/Post.js

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import Image from "./Image.js"
+import UsernamePic from "./UsernamePic.js"
+
+
+class Post extends React.Component{
+  // console.log("inside post")
+  // console.log(props)
+  constructor (props) {
+    super(props);
+    this.state = {
+      likes: props.likes
+    }
+    // console.log('construction')
+  };
+
+  onLikeClick = () => {
+    this.setState((prevState) => ({
+      likes: prevState.likes + 1,
```

Comment:
Right now, every time the like button is clicked, the likes go up 1. We need it to go up by 1, and then down by 1 if we click it again. To do this, you can add a boolean value to state such as "liked" which is set to false. 
And then in your onLikeClick function, we could set the state as such: 
this.state.liked === false ? prevState.likes += 1 : prevState.likes -= 1
This way it either adds 1, or removes 1.